### PR TITLE
Revert "Updating section for git-based chart repositories"

### PR DESCRIPTION
**Pull request by divya-mohan0209**
Reverts rancher/docs#3892 per [this PR](https://github.com/rancher/rancher/pull/36983)

### CODE REVIEW

**Comment by divya-mohan0209** [comment]
@PennyScissors : Reverted the documentation change per request on [this issue](https://github.com/rancher/docs/issues/3864)

File: content/rancher/v2.6/en/helm-charts/_index.md [hunk 1]
```diff
@@ -74,14 +74,11 @@ To add a private CA for Helm Chart repositories:
     [...]
     ```
 
-- **Git-based chart repositories**: You must add a base64 encoded copy of the CA certificate in DER format to the spec.caBundle field of the chart repo, such as `openssl x509 -outform der -in ca.pem | base64 -w0`. Click **Edit YAML** for the chart repo and set, as in the following example:</br>
+- **Git-based chart repositories**: It is not currently possible to add a private CA. For git-based chart repositories with a certificate signed by a private CA, you must disable TLS verification. Click **Edit YAML** for the chart repo, and add the key/value pair as follows:
```
Comment (jtravee) [inline content/rancher/v2.6/en/helm-charts/_index.md:77]:
```suggestion
- **Git-based chart repositories**: It is not currently possible to add a private CA. For git-based chart repositories with a certificate signed by a private CA, you must disable TLS verification. Click **Edit YAML** for the chart repo and add the key/value pair as follows: 
```

Comment (jtravee) [reply content/rancher/v2.6/en/helm-charts/_index.md:77]: Minor nit.

**Review by jtravee — CHANGES_REQUESTED** [review]

**Review by PennyScissors — APPROVED** [review]

**Comment by divya-mohan0209** [comment]
Addressed the review by @jtravee + taking the approval from @PennyScissors, closing this PR.

**Pull request merged**